.github/workflows/rebuild-and-push-images.yml

```
@@ -0,0 +1,62 @@
+name: Rebuild & push images
+on:
+  push:
+    branches:
+      - main
+      - stable
+
+jobs:
+  build-and-push:
+    # To not run in forks
+    if: github.repository == 'packit/packit-service'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - dockerfile: files/docker/Dockerfile
+            image: packit-service
+          - dockerfile: files/docker/Dockerfile.worker
+            image: packit-worker
+          - dockerfile: files/docker/Dockerfile.tests
+            image: packit-service-tests
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # https://stackoverflow.com/a/58035262/2768738
+      - name: Extract branch name
+        shell: bash
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
```

```suggestion
        run: |
            branch=${GITHUB_REF#refs/heads/}
            tag=""
            case $branch in:
                main)
                    tag=stg
                    ;;
                stable)
                    tag=prod
                    ;;
            esac
            echo "::set-output name=branch::$branch"
            echo "::set-output name=tag::$tag"            
```

This should set `steps.extract_branch.outputs.tag`  and the next two steps could be removed.

---

Thanks, exactly what I've been looking for!
Works OK in my fork :rocket: 
Final review, please :-)